images/manageiq-ui-worker/Dockerfile

```diff
@@ -8,9 +8,7 @@ LABEL name="manageiq-ui-worker" \
       summary="ManageIQ user interface worker image"
 
 # Install httpd and remove the existing httpd config from manageiq-appliance
-RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install \
```

Comment:
@simaishi The comment now is kind of misleading?  Maybe something like: ?

"manageiq-ui will install httpd, so remove the existing httpd config from manageiq-appliance"